### PR TITLE
add links to new sample

### PR DESCRIPTION
**Pull request by Rick-Kirkham**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[bf8fcef](https://github.com/OfficeDev/office-js-docs-pr/commits/bf8fcef246bfa95dcd6187845e14c5eb8dfea960)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/auth-with-office-dialog-api.md](https://github.com/OfficeDev/office-js-docs-pr/blob/links-to-new-auth-sample/docs/develop/auth-with-office-dialog-api.md) | :white_check_mark:Succeeded | |
[docs/develop/authorize-to-microsoft-graph-without-sso.md](https://github.com/OfficeDev/office-js-docs-pr/blob/links-to-new-auth-sample/docs/develop/authorize-to-microsoft-graph-without-sso.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C5%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201912052303269698-1474%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=tEEdhN4vf5rPPhvrVkT%2BL4mjrK%2B5z9hFNqaSRKZUNAc%3D&st=2019-12-05T22%3A59%3A41Z&se=2020-01-05T23%3A04%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/develop/auth-with-office-dialog-api.md [hunk 1]
```diff
@@ -79,14 +79,16 @@ These auth-context objects, and the methods that create them, are not usable in
...
+In addition to, or instead of, monolithic "auth context" objects, most libraries provide APIs at a lower level of abstraction that enable your code to create less monolithic helper objects. For example, [MSAL.NET](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki#conceptual-documentation) v. 3.x.x has an API to construct a login URL, and another API that constructs an AuthResult object that contains an access token in a property that is accessible to your code. For examples of MSAL.NET in an Office add-in see: [Office Add-in Microsoft Graph ASP.NET](https://github.com/OfficeDev/PnP-OfficeAddins/tree/master/Samples/auth/Office-Add-in-Microsoft-Graph-ASPNET) and [Outlook Add-in Microsoft Graph ASP.NET](https://github.com/OfficeDev/PnP-OfficeAddins/tree/master/Samples/auth/Outlook-Add-in-Microsoft-Graph-ASPNET). For an example of using [msal.js]() in an add-in, see [Office Add-in Microsoft Graph React](https://github.com/OfficeDev/PnP-OfficeAddins/tree/master/Samples/auth/Office-Add-in-Microsoft-Graph-React).
```
Comment (ElizabethSamuel-MSFT) [inline docs/develop/auth-with-office-dialog-api.md]: Should msal.js link to something?

File: docs/develop/auth-with-office-dialog-api.md [hunk 2]
```diff
@@ -79,14 +79,16 @@ These auth-context objects, and the methods that create them, are not usable in
...
 
 For more information about authentication and authorization libraries, see [Microsoft Graph: Recommended libraries](authorize-to-microsoft-graph-without-sso.md#recommended-libraries-and-samples) and [Other external services: Libraries](auth-external-add-ins.md#libraries).
 
 ## Samples
 
-- [Office Add-in Microsoft Graph ASP.NET](https://github.com/OfficeDev/PnP-OfficeAddins/tree/master/Samples/auth/Office-Add-in-Microsoft-Graph-ASPNET): An ASP.NET based add-in (Excel, Word, or PowerPoint) that uses the MSAL.NET library to login and get an access token for Microsoft Graph data.
+- [Office Add-in Microsoft Graph ASP.NET](https://github.com/OfficeDev/PnP-OfficeAddins/tree/master/Samples/auth/Office-Add-in-Microsoft-Graph-ASPNET): An ASP.NET based add-in (Excel, Word, or PowerPoint) that uses the MSAL.NET library, and the Authorization Code Flow, to login and get an access token for Microsoft Graph data.
```
Comment (ElizabethSamuel-MSFT) [inline docs/develop/auth-with-office-dialog-api.md]:
```suggestion
- [Office Add-in Microsoft Graph ASP.NET](https://github.com/OfficeDev/PnP-OfficeAddins/tree/master/Samples/auth/Office-Add-in-Microsoft-Graph-ASPNET): An ASP.NET based add-in (Excel, Word, or PowerPoint) that uses the MSAL.NET library and the Authorization Code Flow to log in and get an access token for Microsoft Graph data.
```

File: docs/develop/auth-with-office-dialog-api.md [hunk 3]
```diff
@@ -79,14 +79,16 @@ These auth-context objects, and the methods that create them, are not usable in
...
 - [Outlook Add-in Microsoft Graph ASP.NET](https://github.com/OfficeDev/PnP-OfficeAddins/tree/master/Samples/auth/Outlook-Add-in-Microsoft-Graph-ASPNET): Just like the one above, but the Office application is Outlook.
+- [Office Add-in Microsoft Graph React](https://github.com/OfficeDev/PnP-OfficeAddins/tree/master/Samples/auth/Office-Add-in-Microsoft-Graph-React): A NodeJS based add-in (Excel, Word, or PowerPoint) that uses the msal.js library, and the Implicit Flow, to login and get an access token for Microsoft Graph data.
```
Comment (ElizabethSamuel-MSFT) [inline docs/develop/auth-with-office-dialog-api.md]:
```suggestion
- [Office Add-in Microsoft Graph React](https://github.com/OfficeDev/PnP-OfficeAddins/tree/master/Samples/auth/Office-Add-in-Microsoft-Graph-React): A NodeJS based add-in (Excel, Word, or PowerPoint) that uses the msal.js library and the Implicit Flow to log in and get an access token for Microsoft Graph data.
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[70fe908](https://github.com/OfficeDev/office-js-docs-pr/commits/70fe908ea4bc9a50d7ae4b6fcc75868b34eb349d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/auth-with-office-dialog-api.md](https://github.com/OfficeDev/office-js-docs-pr/blob/links-to-new-auth-sample/docs/develop/auth-with-office-dialog-api.md) | :white_check_mark:Succeeded | |
[docs/develop/authorize-to-microsoft-graph-without-sso.md](https://github.com/OfficeDev/office-js-docs-pr/blob/links-to-new-auth-sample/docs/develop/authorize-to-microsoft-graph-without-sso.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C5%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201912052319485542-1474%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=ogs2yR2OYAsLikPClbV6QqwZJSSIUGhYI1pxQq9Jd2c%3D&st=2019-12-05T23%3A16%3A01Z&se=2020-01-05T23%3A21%3A01Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[b244e70](https://github.com/OfficeDev/office-js-docs-pr/commits/b244e7041dabfa6bb251a7128c55f6a8f5fefbf1)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/auth-with-office-dialog-api.md](https://github.com/OfficeDev/office-js-docs-pr/blob/links-to-new-auth-sample/docs/develop/auth-with-office-dialog-api.md) | :white_check_mark:Succeeded | |
[docs/develop/authorize-to-microsoft-graph-without-sso.md](https://github.com/OfficeDev/office-js-docs-pr/blob/links-to-new-auth-sample/docs/develop/authorize-to-microsoft-graph-without-sso.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C5%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201912052320212240-1474%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=2dSBpt7ZlXnXPzNXvFhcYZjS1xtMMmdZ8yOoafeeqLw%3D&st=2019-12-05T23%3A17%3A18Z&se=2020-01-05T23%3A22%3A18Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**